### PR TITLE
Drop exception handler in for tag to reduce bytecode size

### PR DESCRIPTION
**Pull request by e5l**

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -68,80 +68,37 @@ publishing {
...
-        }
     }
 }
 
 repositories {
     mavenCentral()
 }
 
+
+
```
Comment (rsinukov) [inline build.gradle.kts:79]: unnecessary empty lines

**Comment by languitar** [comment]
Seems this change got into the 0.9.0 release without being somehow marked as a breaking change. For us, this has effectively broken error handling in a ktor project. Is this expected?

**Comment by e5l** [comment]
@languitar, yep, it is. Could you share the case of how exceptions were handled in your project?

**Comment by languitar** [comment]
What was broken was the integration with kotlinx html provided by ktor themselves. But it seems the vanished method is handled on main already: https://youtrack.jetbrains.com/issue/KTOR-6124/Not-compatible-with-kotlinx-html-0.9.1

Is it intended that breaking changes appear in this project without major version bumps?

**Comment by e5l** [comment]
Yep, it is before the stable `1.0.0`release

**Pull request merged**